README.md

```diff
@@ -32,6 +32,9 @@ specification.
 * `ring` (default): use the *ring* crate as the crypto backend
 * `aws-lc-rs`: use the aws-lc-rs crate as the crypto backend
 * `fips`: enable the aws-lc-rs crate's FIPS-compliant mode
+* `x509-parser`: enable creating `CertificateIdentifier` instances for ARI from
```

Comment:
Maybe "extracting `CertificateIdentifier` values for ARI from certificates"?

Reply:
Reworded.